keyboards/signum/3_1/3_1.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "quantum.h"
+
+// LAYOUT is for there being 2 thumb cluster keys and 3 keys below alphas
+#define LAYOUT_2_thumb_3_sub_alpha( \
+	K000, K001, K002, K003, K004, K504,      K505, K005, K006, K007, K008, K009, \
+	K100, K101, K102, K103, K104, K503,      K506, K105, K106, K107, K108, K109, \
+	K200, K201, K202, K203, K204, K502,      K507, K205, K206, K207, K208, K209, \
+	K300, K301, K302, K303, K304, K501,      K508, K305, K306, K307, K308, K309, \
+	K400, K401, K402, K403, K404, K500,      K509, K405, K406, K407, K408, K409  \
```

Comment:
```suggestion
#define LAYOUT_2_thumb_3_sub_alpha(
	K000, K001, K002, K003, K004, K504,      K505, K005, K006, K007, K008, K009,
	K100, K101, K102, K103, K104, K503,      K506, K105, K106, K107, K108, K109,
	K200, K201, K202, K203, K204, K502,      K507, K205, K206, K207, K208, K209,
	K300, K301, K302, K303, K304, K501,      K508, K305, K306, K307, K308, K309,
	K400, K401, K402, K403, K404, K500,      K509, K405, K406, K407, K408, K409
```

Don't need trailing slashes on parameters, only the content.